requirements.txt

```diff
@@ -1,6 +1,6 @@
 aiofiles>=0.8.0
 async_lru~=1.0
-galaxy_importer>=0.4.1
+galaxy_importer>=0.4.5
```

Comment:
their release failed: https://github.com/ansible/galaxy-importer/runs/6479389918?check_suite_focus=true

Reply:
galaxy-importer 0.4.5 is now on pypi